index.bs

```diff
@@ -2072,6 +2072,8 @@ When registering a new credential, represented by a {{AuthenticatorAttestationRe
 
 15. If verification of the attestation statement failed, the [=[RP]=] MUST fail the registration ceremony.
 
+16. Store the public key and the signature counter for this authenticator.
+
```

Comment:
There should be a line above (probably around decoding AuthenticatorData) about where the signature counter comes from.

Reply:
Done.